docs/library/disclosure/accordion.md

```diff
@@ -0,0 +1,384 @@
+---
+components:
+    - rx.components.radix.primitives.AccordionRoot
+    - rx.components.radix.primitives.AccordionHeader
+    - rx.components.radix.primitives.AccordionContent
+---
+
+```python exec
+import reflex as rx
+from reflex.components.radix.themes.components import *
+from reflex.components.radix.themes.layout import *
+from reflex.components.radix.themes.typography import *
+from reflex.components.radix.primitives import *
+from reflex.components.core import *
+from reflex.components.radix.themes import *
+```
+
+# Accordion
+
+An accordion is a vertically stacked set of interactive headings that each reveal an associated section of content.
+The accordion component us made up of `accordion` which is the root of the component and takes in an `accordion_item`
+which contains all the contents of the collapsible section.
+
+## Basic Example
+
+```python demo
+accordion(
+    accordion_item("First Item", "The first accordion item's content", font_size="3em"),
```

Comment:
I think the default padding on this is not very good - maybe we need to improve the high level API for this